### PR TITLE
User Login Function

### PR DESCRIPTION
**Pull request by 2kama**
User Login Function

### CODE REVIEW

File: src/store/sagas/handlers/user.js [hunk 1]
```diff
@@ -29,14 +26,23 @@ export function* handleRegisterUser(action) {
...
+        console.log(alertData, err.message)
         yield put(setAlert(alertData))
         yield put(enableButton())
+    }
+}
+
+export function* handleLoginUser(action) {
+    try {
+
+        yield call(requestLoginUser(action.userData))
```
Comment (DeveloperX104) [inline src/store/sagas/handlers/user.js]: THere should be a `bind` method on functions that takes in parameter, to make it work without any error or warning.

Comment (2kama) [reply src/store/sagas/handlers/user.js:45]: Please explain...

Comment (2kama) [reply src/store/sagas/handlers/user.js]: Thanks .... Already added this... that took care of the error I was getting even on success.

Comment (Babatunde13) [reply src/store/sagas/handlers/user.js]: Yhhh.

**Comment by Babatunde13** [comment]
Also, I noticed that the button isn't disabled even though the code shows that it should be.
The PR I opened on sign in disables the button but I'm unable to figure out what I did differently there.

File: src/store/sagas/handlers/user.js (hunk 1 above)
Comment (2kama) [reply src/store/sagas/handlers/user.js]: Do you feel this can be approved now?

**Review by DeveloperX104 — APPROVED** [review]

**Pull request merged**